enhancements/olm/platform-operators.md

```diff
@@ -0,0 +1,609 @@
+
+### Plain Bundle Format
+
+A "plain" bundle format is OLM's newest bundle format that supports packaging arbitrary Kubernetes resources. This bundle format can also be referred to as the plain+v0 bundle format.
+
+### Resolveset Bundle Format
+
+A "resolveset" is an OLM bundle format that expresses the set of bundles that need to exist on the cluster in order to satisfy consumer constraints. While this resolveset bundle format is still in the work-in-progress stage, we can currently define this format as a YAML stream of individual BundleDeployment manifests.
+
+### Bundle/BundleDeployment API(s)
+
+A "BundleDeployment" is an OLM API that's responsible for managing OLM's first-class Bundle API, which represents content that needs to be made available to other consumers on the cluster. Both of these APIs are cluster-scoped resources.
+These APIs are analogous to the Deployment/PodSpec resources seen throughout core Kubernetes. The BundleDeployment API allows users to specify a Bundle template, and that template contains the desired specification
+of a Bundle resource. The management of a Bundle and BundleDeployment resources is delegated to individual "provisioner" implementations.
+
+### Provisioner
+
+A "provisioner" is a controller that understands the Bundle and BundleDeployment APIs. Individual provisioner implementations use ingress-style "class" name identifiers to distinguish between the BundleDeployments and Bundles resources that it needs to manage. Each provisioner implementation requires the definition of a unique provisioner IDs.
+This provisioner pattern helps abstract the variations of bundle formats that are defined throughout the Kubernetes landscape. Future provisioner implementations may manage helm charts, crossplane packages, and OLM bundle formats.
+
+Both the Bundle and BundleDeployment APIs require a unique provisioner ID to be specified. This enables higher level systems (e.g. a PO manager) to generate BundleDeployments that referencing different unique provisioner IDs. This is useful for components that want more control over how to source, and apply, Bundle contents.
+
+## Proposal
+
+### Introducing a "PlatformOperator" CRD
+
+This first phase will introduce a new, top-level "PlatformOperator" cluster-scoped CRD. While admins today could include OLM manifests before cluster installation, this approach lacks the ability to influence the cluster's lifecycle.
+This new CRD will bring CVO awareness of these manifests, and help pave the way for adding a first-class UX (e.g. via the installer's install-config API) for configuring POs in future phases.
+
+Given this is an openshift-only API, this proposal recommends introducing the CRD into the openshift/api repository under the `platform.openshift.io/v1alpha1` API group. During phase 0, this resource will only contain a single, required `spec.PackageName` field. This field will be used to configure the desired OLM-based Operator package name.
```

Comment:
nit: camelcase? 
```suggestion
Given this is an openshift-only API, this proposal recommends introducing the CRD into the openshift/api repository under the `platform.openshift.io/v1alpha1` API group. During phase 0, this resource will only contain a single, required `spec.packageName` field. This field will be used to configure the desired OLM-based Operator package name.
```

Reply:
Updated in https://github.com/openshift/enhancements/pull/1170/commits/bc14b4537125d43153cc452cd686e02cf96630b2.